src/ducks/modules/protocol/utils/dialogs.js

```diff
@@ -26,7 +25,30 @@ export const invalidAssetErrorDialog = (e, filePath) => {
   return dialogActions.openDialog({
     type: 'UserError',
     title: `Error: ${path.basename(filePath)} is not formatted correctly`,
-    error: e,
+    message: friendlyMessage,
+  });
+};
+
+export const unsupportedAssetErrorDialog = () => {
+  const friendlyMessage = (
+    <React.Fragment>
+      <p>
+        The file you attmpted to import is not in a format supported by Network
```

Comment:
Should be "attempted", not "attmpted".

Reply:
> This takes away the "type" handling. Adding a "network data file" on a side panel for a "Name generator" interface now allows any valid type of file to be dropped, including images, video, csv, etc.

Good point. I'll think about this a bit more.